app/control.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package app
```

Comment:
Can we move this to `model`? I'm fearing a potential import cycle in the future.

Reply:
yes, we can. but for exactly the reason of import cycles we cant move recorder. are you fine with having control.go in model and rest in the app?